common/api/itwin-registry-client.api.md

```diff
@@ -29,6 +29,10 @@ export interface ITwinAccess {
 export class ITwinAccessClient extends WsgClient implements ITwinAccess {
     constructor();
     getAll(accessToken: AccessToken, arg?: ITwinQueryArg): Promise<ITwin[]>;
+    // @internal (undocumented)
+    protected getUrlSearchKey(): string;
+    // (undocumented)
+    static readonly searchKey: string;

```

Comment:
Also, missed where the search key was but can you also remove that from the client?